src/chainer.test.js

```diff
@@ -1 +1,50 @@
-const { chainer } = require('./chainer');
+'use strict';
+
+describe('chainer', () => {
+  const { chainer } = require('./chainer');
+
+  it(`should correctly chain functions and return the expected result`, () => {
+    const addOne = (x) => x + 1;
+    const multiplyByTwo = (x) => x * 2;
+    const subtractThree = (x) => x - 3;
+
+    const chainedFunctions = chainer([addOne, multiplyByTwo, subtractThree]);
+
+    expect(chainedFunctions(5)).toEqual(9);
+  });
+
+  it(`should correctly chain functions and return the expected'
+  + 'result for 0 value`, () => {
+    const addOne = (x) => x + 1;
+    const multiplyByTwo = (x) => x * 2;
+    const subtractThree = (x) => x - 3;
+
+    const chainedFunctions = chainer([addOne, multiplyByTwo, subtractThree]);
+
+    expect(chainedFunctions(0)).toEqual(-1);
+  });
+
+  it(`should correctly chain functions and return the expected'
+  + 'result for null value`, () => {
+    const addOne = (x) => x + 1;
+    const multiplyByTwo = (x) => x * 2;
+    const subtractThree = (x) => x - 3;
+
+    const chainedFunctions = chainer([addOne, multiplyByTwo, subtractThree]);
+
+    expect(chainedFunctions(null)).toEqual(-1);
```

Comment:
The test result for null value is incorrect since addOne(null) will return 1, multiplyByTwo(1) will return 2 and subtractThree(2) will return -1. So the expected value should be -1, not NaN.